pkgs/by-name/qb/qbittorrent-exporter/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGo125Module,
+  nix-update-script,
+}:
+buildGo125Module rec {
+  pname = "qbittorrent-exporter";
+  version = "1.11.0";
+
+  src = fetchFromGitHub {
+    owner = "martabal";
+    repo = pname;
```

```suggestion
    repo = "qbittorrent-exporter";
```
Not a big issue, but that's discouraged: https://github.com/NixOS/nixpkgs/issues/277994